modules/lib/src/main/resources/assets/js/app/view/ContentItemPreviewPanel.ts

```diff
@@ -110,7 +111,15 @@ export class ContentItemPreviewPanel
         this.fetchPreviewForPath(contentSummary);
     }
 
+    public isRenderable(): Q.Promise<boolean> {
```

Comment:
This tells if currently displayed item is renderable because is renderable request is now dropped, will rename to isItemRenderable.
Public in case other components want to know if item is renderable.